### ISSUE TITLE
Angular Universal problems

### ISSUE THREAD

**Issue by rimlin** [Tsmith18256/ng2-inline-href#9]
In using Angular Universal i see the problem: `ERROR ReferenceError: window is not defined`
Maybe instead of using `window` use [Location service](https://angular.io/docs/ts/latest/api/common/index/Location-class.html) or something similar?

**Comment by Tsmith18256** [comment]
Hmmm, I haven't used Angular Universal. I'll familiarize myself with it over the weekend and get this fixed. Thanks for the link!

**Comment by Tsmith18256** [comment]
Hey @rimlin, I have changed the directive to use the location service, but I didn't have time to set up an Angular Universal project. The new changes work correctly in the browser, can you download the latest version from NPM (`ng-inline-href@2.1.0`) and confirm if it works with your Angular Universal project?

**Comment by Tsmith18256** [comment]
Hey again @rimlin, since I haven't heard from you yet, I'm gonna close this issue. If you try the latest version and still have an issue, just let me know.

**Comment by rimlin** [comment]
Hi @Tsmith18256, i apologize for not answering, I've been busy last week. Looked at the changes - work well, but in server console output this error:

```
ERROR TypeError: this.element.nativeElement.setAttributeNS is not a function
    at InlineHrefDirective.ngOnInit (D:\dist\server.js:164000:40)
```

I think in server part `nativeElement` cannot use `setAttributeNS` function. You can check [Renderer2.setAttribute](https://angular.io/api/core/Renderer2#setAttribute) function to set property at element, try this:

```
ngOnInit(): void {
    const includeHash = false;
    const fixedHref = this.location.path(includeHash) + this.inlineHref;

    if (this.element.nativeElement.tagName === InlineHrefDirective.SVG_USE_TAG) {
      this.renderer.setAttribute(this.element.nativeElement, InlineHrefDirective.HREF_ATTR, fixedHref, InlineHrefDirective.XLINK_NS);
    } else {
      this.element.nativeElement.href = fixedHref;
    }
}
```

If you want, i can fix this issue at this week.

**Comment by Tsmith18256** [comment]
Hey @rimlin, really sorry for kinda disappearing on this, it's been a pretty crazy month for me. If you want to, I will gladly accept a PR for this issue; otherwise, I should have time to look at it myself in the next day or two.

**Comment by Tsmith18256** [comment]
Fixed in 2.1.1, which is live on NPM finally! Thanks for your help
